resources/js/controllers/fields/relation_controller.js

```diff
@@ -16,6 +16,7 @@ export default class extends Controller {
         const key = this.data.get('key');
         const scope = this.data.get('scope');
         const append = this.data.get('append');
+        const columns = this.data.get('columns');
```

Comment:
I think the name `columns` does not convey the whole meaning.
Can we reimagine them to `searchColumns`?

Reply:
I rename method ```searchIn``` to ```searchColumns``` too.